### PR TITLE
Move to jetty9 since jetty8 is EOL.

### PR DESCRIPTION
**Pull request by aquamatthias**
Bump version of underlying libs.

### CODE REVIEW

**Comment by aquamatthias** [comment]
And yes I can not believe I do this either...

File: project/build.scala [hunk 1]
```diff
@@ -1,13 +1,12 @@
 import com.amazonaws.auth.InstanceProfileCredentialsProvider
-import com.amazonaws.auth.profile.ProfileCredentialsProvider
+import com.typesafe.sbt.SbtScalariform._
 import ohnosequences.sbt.SbtS3Resolver
+import ohnosequences.sbt.SbtS3Resolver.{S3Resolver, s3, s3resolver}
+import org.scalastyle.sbt.ScalastylePlugin.{Settings => styleSettings}
```
Comment (gkleiman) [inline project/build.scala]: We use spaces around the curly braces everywhere else.

File: src/main/scala/mesosphere/chaos/http/HttpModule.scala [hunk 2]
```diff
@@ -146,7 +150,7 @@ class HttpModule(conf: HttpConf) extends AbstractModule {
     resourceCacheControlHeader foreach handler.setCacheControl
     handler.setWelcomeFiles(welcomeFiles)
     handler.setResourceBase(conf.assetsUrl().toExternalForm)
-    handler.setAliases(true) // Enables use of relative paths
+    // handler.setAliases(true) // TODO: Enables use of relative paths
```
Comment (gkleiman) [inline src/main/scala/mesosphere/chaos/http/HttpModule.scala]: I'd remove this line instead of commenting it out.

**Comment by aquamatthias** [comment]
@gkleiman Incorporated your feedback.

**Comment by gkleiman** [comment]
Thanks!

**Pull request merged**